### PR TITLE
Knitting a release

### PR DESCRIPTION
**Pull request by rtibbles**
## Summary

A (hopefully) release compatible version of #796.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -83,7 +84,7 @@
     "mkdirp": "0.5.1",
     "mocha": "2.5.3",
     "phantomjs-polyfill-find": "ptim/phantomjs-polyfill-find",
-    "phantomjs-prebuilt": "2.1.7",
+    "phantomjs-prebuilt": "^2.1.14",
```
Comment (indirectlylit) [inline package.json:87]: Inserted a `^`, which is inconsistent with the rest of the dependencies.

Now that we have lock files, perhaps best-practice has changed? (E.g. all dependencies could be brought back to [carat ranges](https://docs.npmjs.com/misc/semver#caret-ranges-123-025-004), and we can pull upgrades early in each dev cycle.)

Comment (rtibbles) [reply package.json:87]: Yeah, yarn actually inserted this caret, not I. So, I imagine we can phase them out as we upgrade packages in the release cycle upgradathon!

**Comment by rtibbles** [comment]
IT FINALLY PASSED THE GODDAMN TESTS!

**Comment by indirectlylit** [comment]
hooray! 🎉 

Ready for review?

**Comment by rtibbles** [comment]
Yes, should be good to go (barring any horrible things you find in review).

File: kolibri/core/webpack/hooks.py [hunk 2]
```diff
@@ -139,20 +139,24 @@ def webpack_bundle_data(self):
...
+                "plugin_path": os.path.dirname(self.build_path),
+                "stats_file": self.stats_file,
+                "events": self.events,
+                "once": self.once,
+                "static_url_root": getattr(django_settings, 'STATIC_URL'),
+                "locale_data_folder": os.path.join(getattr(django_settings, 'LOCALE_PATHS')[0], 'en', 'LC_FRONTEND_MESSAGES'),
+                "version": self.version,
+            }
+        else:
+            logger.warn("{src_file} not found for plugin {name}.".format(src_file=self.src_file, name=self.unique_slug))
```
Comment (aronasorman) [inline kolibri/core/webpack/hooks.py:159]: So we don't return anything if the src file doesn't exist. What exactly is this src file?

File: kolibri/plugins/coach_tools/yarn.lock [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (aronasorman) [inline kolibri/plugins/coach_tools/yarn.lock:1]: What does this lock file do? Doesn't seem like there's anything important specified here.

File: kolibri/core/webpack/hooks.py (hunk 2 above)
Comment (rtibbles) [reply kolibri/core/webpack/hooks.py:159]: It's the entry file for the webpack bundle.

File: kolibri/plugins/coach_tools/yarn.lock (hunk 3 above)
Comment (rtibbles) [reply kolibri/plugins/coach_tools/yarn.lock:1]: It locks any dependencies for this plugin. This plugin has no separate dependencies, hence it is empty.

**Pull request merged**